ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -51,7 +51,7 @@ RHODS Operator Should Be Uninstalled
 Uninstall RHODS In Self Managed Cluster Using CLI
   [Documentation]   UnInstall rhods on self-managedcluster using cli
   Clone OLM Install Repo
-  ${return_code}    Run and Watch Command    cd ${EXECDIR}/${OLM_DIR} && ./cleanup.sh -t operator    timeout=10 min
+  ${return_code}    Run and Watch Command    cd ${EXECDIR}/${OLM_DIR} && ./cleanup.sh -t operator -a "authorino serverless servicemesh"    timeout=10 min
```

Comment:
will it also clean crd ?

Reply:
Looking at the changes at olminstall it seems that yes, the crd's are deleted